### ISSUE TITLE
[White Label - Audiocard]: In High contrast Aquatic mode,  Luminsoity ratio of Play, Voice and More icons which are less than required ratio of 3:1.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5710]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users with low vision or users who rely on High Contrast Aquatic mode to improve visual clarity may struggle to identify or locate the Play, Voice, and More icons due to insufficient contrast. Poor visibility of these icons can lead to difficulty understanding available actions, reduced efficiency, and potential inability to interact with essential controls. 

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Contrast ratio

### How to reproduce the issue?

Pre-Requisite:
Go to Settings--> Accessibility--> Contrast Themes--> Apply Aquatic mode.

Repro Steps
1. Open URL Web Chat in EDGE browser.
2. Webchat page will appear.
3. Navigate to edit field and provide query 'Audiocard'.
4. Response will appear.
5. Observe an issue that In High contrast Aquatic mode, Luminosity ratio of Play, Voice and More icons which are less than required ratio of 3:1.

### What do you expect?

In High Contrast Aquatic mode, all non‑text UI icons such as Play, Voice, and More must meet or exceed the minimum luminosity contrast ratio of 3:1 against adjacent colors.

### What actually happened?

In High contrast Aquatic mode, Luminosity ratio of Play, Voice and More icons which are less than required ratio of 3:1.

### Do you have any screenshots or recordings to repro the issue?

<img width="1932" height="1413" alt="Image" src="https://github.com/user-attachments/assets/6d2b31fd-d0d1-4dae-8de1-5c2d0a11a8d3" />

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

### Adaptive Card JSON

```json

```
